src/Dev-7/checkerDate/CheckerInputDate.java

```
@@ -0,0 +1,22 @@
+package checkerDate;
+
+import inputer.InputDataAndTime;
+
+/**
+ * Check input data for null
+ */
+public class CheckerInputDate {
+
+
+    /**
+     * Check entry data
+     * @return If entry data is null retun
+     */
+    public boolean checker() {
+        InputDataAndTime inputDataAndTime = new InputDataAndTime();
+        if (inputDataAndTime.getDate() == "") {
+            System.out.println("You don't enter anything!");
```

throw exception instead of just hiding error